routes/api.js

```diff
@@ -0,0 +1,43 @@
+const { Router } = require('express')
+const router = Router ()
+
+// Auth
+router.post('/auth/signup', require('../controllers/api/auth/signup'))
+router.post('/auth/login', require('../controllers/api/auth/login'))
+router.delete('/auth/logout', require('../controllers/api/auth/logout'))
+
+// Public products
+router.get('/products'), require('../controllers/api/products/index')
+router.get('/products/:id'), require('../controllers/api/products/show')
+
+// My orders - for payment
+router.get('/my/orders', require('../controllers/api/my/orders'))
+router.post('/my/orders', require('../controllers/api/my/orders'))
+router.delete('/my/orders', require('../controllers/api/my/orders'))
+router.post('/my/orders/:id/pay', require('../controllers/api/my/pay'))
+router.put('/my/orders', require('../controllers/api/my/orders'))
+
```

Comment:
may need to add:
require(... my/orders/new)
require(... my/orders/update)
require(... my/orders/destroy) etc